scripts/src/tests/helpers.ts

```diff
@@ -1,17 +1,21 @@
 import { getManager } from "typeorm";
 import * as StellarSdk from "stellar-sdk";
 
-import { generateId } from "../utils/utils";
+import { generateId, readKeysDir, random } from "../utils/utils";
 import { Asset, Offer } from "../models/offers";
 import { User, AuthToken } from "../models/users";
-import { Application, ApplicationConfig } from "../models/applications";
+import { Application, ApplicationConfig, StringMap } from "../models/applications";
```

Comment:
please move the `StringMap` thing to another file, maybe the `utils/utils` module or even create a new `utils/types` module?

Reply:
There are many uses of `StringMap`, I can do it in separate PR